bedrock/products/views.py

```diff
@@ -40,11 +40,13 @@ def vpn_available(request):
 def relay_available(product, request):
     country = get_country_from_request(request)
     if product == "relay-bundle":
-        country_list = settings.VPN_COUNTRY_CODES
+        country_list = settings.RELAY_PLANS_BY_COUNTRY_AND_LANGUAGE["bundle"]
     elif product == "relay-phone":
-        country_list = settings.RELAY_PHONE_COUNTRY_CODES
+        country_list = settings.RELAY_PLANS_BY_COUNTRY_AND_LANGUAGE["phone"]
+    elif product == "relay-email":
+        country_list = settings.RELAY_PLANS_BY_COUNTRY_AND_LANGUAGE["email"]
     else:
-        country_list = settings.RELAY_EMAIL_COUNTRY_CODES
+        raise Exception("Unrecognized product passed to relay_available()")
```

Comment:
This will cause a 500 server error if encountered. I think that's okay since we don't know what to do with this request if it doesn't match a product here. And shouldn't happen ever. Just pointing it out just in case.